examples/spring-boot-with-tekton-example/readme.md

```diff
@@ -0,0 +1,59 @@
+# Spring Boot on Kubernetes 
+
+The purpose of this example is to demonstrate the following:
+
+- How you can use the kubernetes-spring-stater.
+- How dekorate detects that this is a web app and automatically configures services and probe.
```

Comment:
```suggestion
- How Dekorate detects that this is a web app and automatically configures services and probe.
```

Reply:
This keeps on comming up again and again. For me it always felt like `dekorate` is how it should be, its even reflected in the new logo :-)

Reply:
Makes sense, but the case should be consistent in the readme